exporters/failure/collector_azure_devops.py

```diff
@@ -0,0 +1,174 @@
+            return True
+        return str(priority) in self.work_item_priority
+
+    def filter_by_project(self, project: str) -> bool:
+        if not self.projects:
+            return True
+        return project in self.projects
+
+    def get_app_name(self, work_item: WorkItem) -> str:
+        try:
+            labels: str = work_item.fields["System.Tags"]
+            labels = labels.split("; ")
+
+            label_text = self.app_label + "="
+
+            for label in labels:
+                if label_text in label:
+                    return label.replace(label_text, "")
+            return "unknown"
+        except KeyError:
+            return "unknown"
+
+    def search_issues(self) -> list[TrackerIssue]:
+        """
+        To maintain consistency, we call this method `search_issues`. An
+        `issue` in Azure DevOps is called `work item`.
+        """
+        production_work_items = []
+        for work_item in self.get_work_items():
+            is_production_bug = (
+                self.filter_by_type(work_item.fields["System.WorkItemType"])
+                and self.filter_by_priority(
+                    work_item.fields["Microsoft.VSTS.Common.Priority"]
+                )
+                and self.filter_by_project(work_item.fields["System.TeamProject"])
+            )
+
+            if is_production_bug:
+                created_at = work_item.fields["System.CreatedDate"]
+                work_item_id = work_item.id
+                title = work_item.fields["System.Title"]
+
+                created_tz = parse_assuming_utc(created_at, _DATETIME_FORMAT)
+                created_ts = second_precision(created_tz).timestamp()
+
+                try:
+                    resolved_at = work_item.fields["Microsoft.VSTS.Common.ClosedDate"]
+                    resolution_tz = parse_assuming_utc(resolved_at, _DATETIME_FORMAT)
+                    resolution_ts = second_precision(resolution_tz).timestamp()
+
+                    logging.debug(
+                        "Found production incident closed: {}, {}: {}".format(
+                            resolved_at,
+                            work_item_id,
+                            title,
+                        )
+                    )
+                except KeyError:
+                    logging.debug(
+                        "Found production incident opened: {}, {}: {}".format(
+                            created_at,
+                            work_item_id,
+                            title,
+                        )
+                    )
+                    resolution_ts = None
+
+                tracker_issue = TrackerIssue(
+                    str(work_item_id),
+                    created_ts,
+                    resolution_ts,
+                    self.get_app_name(work_item),
+                )
+                production_work_items.append(tracker_issue)
+        if not production_work_items:
+            # TODO should be warning?
```

Comment:
No it should not be warning, it's perfectly fine to not have issues in the project.